jiant/preprocess.py

```diff
@@ -614,3 +671,25 @@ def add_wsj_vocab(vocab, data_dir, namespace="tokens"):
     for line in wsj_tokens.readlines():
         vocab.add_token_to_namespace(line.strip(), namespace)
     log.info("\tAdded WSJ vocabulary from %s", wsj_tokens)
+
+
+class TaskModulator(object):
+    """ A task modulator describes everything the task process needs to know about the model
```

Comment:
Perhaps change the documentation to be "This class holds parts of preprocessing that is model-specific."

Reply:
okay

Reply:
> This looks good to me. Do experiments with GPT1 match the old implementation?

I don't know yet. I'll let you know when results come out.